public/gameInfo.php

```diff
@@ -963,7 +963,7 @@ function ResetProgress() {
                     echo "<input type='hidden' name='genre' value='" . attributeEscape($genre) . "'>";
                     echo "<input type='hidden' name='release' value='" . attributeEscape($released) . "'>";
                     echo "<div class='md:grid grid-cols-[180px_1fr_100px] gap-1 items-center mb-1'>";
-                    echo "<label for='guide_url'>Guide URL</label><input type='url' name='guide_url' id='guide_url' value='" . attributeEscape($guideURL) . "' class='w-full'>";
+                    echo "<label for='guide_url' class='cursor-help italic' title='Must be from https://github.com/RetroAchievements/guides'>Guide URL<sup>*</sup></label><input type='url' name='guide_url' id='guide_url' value='" . attributeEscape($guideURL) . "' class='w-full'>";
```

Comment:
Why does this get to be italic? None of the other fields are.

Reply:
I was hoping to draw some kind of connection that there's a tooltip attached to the label.

Reply:
The cursor-help is probably sufficient.

The only other place I can think of where we do something like this is the report ticket page, but those pop up a window instead of just a tooltip.

Reply:
Resolved in https://github.com/RetroAchievements/RAWeb/pull/1553/commits/4700b3d33325f06fb2ed051461cda9132606a936